com.github.hopsan.Hopsan.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>se.hopsan.Hopsan</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0 AND GPL-3.0</project_license>
+  <name>Hopsan</name>
+  <summary>A modelling and simulation tool for fluid power and mechatronic systems</summary>
+
+  <description>
+    <p>
+    Hopsan is a modelling and simulation environment for fluid and mechatronic systems. It was originally developed for simulation of fluid power systems, but has also been extended with support for other domains such as electric power, mechanics, flight and vehicle dynamics.
+    </p>
+    <p>
+    Hopsan uses bi-directional delay lines, (or transmission line elements TLM) to connect physical component models, but also supports signal flow modelling with common mathematical operators and functions.
+    </p>
+  </description>
+
+  <content_rating type="oars-1.0" />
+
+  <launchable type="desktop-id">com.github.hopsan.Hopsan.desktop</launchable>
+
+  <url type="homepage">https://liu.se/en/research/hopsan</url>
+  <url type="bugtracker">https://github.com/Hopsan/hopsan/issues</url>
+  <url type="help">https://flumes.iei.liu.se/hopsan/docs/latest/html</url>
+  <update_contact>https://github.com/Hopsan/hopsan/issues</update_contact>
+
```

Comment:
You could add a donate url if you have one. Also the developer_name tag could be useful.

Reply:
We do not take donations but added developer_name as suggested. I added two such fields, I hop that it will work